Demo/PowerSyncExample/PowerSync/SupabaseConnector.swift

```diff
@@ -4,11 +4,45 @@ import Supabase
 import PowerSync
 import AnyCodable
 
+private struct PostgresFatalCodes {
+    /// Postgres Response codes that we cannot recover from by retrying.
+    static let fatalResponseCodes: [String] = [
```

Comment:
Can we recover from deadlocks or serialization failures?

Reply:
Not sure